DeskReserve/Controllers/RoomsController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using DeskReserve.Data.DBContext;
+using DeskReserve.Data.DBContext.Entity;
+
+namespace DeskReserve.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RoomsController : ControllerBase
+    {
+        private readonly ApplicationDbContext _context;
+
+        public RoomsController(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Rooms
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Room>>> GetRooms()
+        {
+            return await _context.Rooms.ToListAsync();
+        }
+
+        // GET: api/Rooms/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Room>> GetRoom(Guid id)
+        {
+            var room = await _context.Rooms.FindAsync(id);
+
+            if (room == null)
+            {
+                return NotFound();
+            }
+
+            return room;
```

Comment:
Always specify the return type , in this situation return Ok(room), Be consistent in all controller method